cmd/tailscaled/tailscaled.go

```diff
@@ -160,7 +161,12 @@ func main() {
 		log.Fatalf("--socket is required")
 	}
 
-	if err := run(); err != nil {
+	err := run()
+
+	// Remove file sharing from Windows shell (noop in non-windows)
+	osshare.SetFileSharingEnabled(false)
```

Comment:
fix this staticcheck warning. Maybe make the noop version have a side effect, like:

```go
// Some dummy side effect to shut up staticcheck SA4017.
select {
case <-context.Background().Done():
default:
}
```